### ISSUE TITLE
RFE: Add MapTextColor to the Client Settings Advanced Options

### ISSUE THREAD

**Issue by Inglonias** [MegaMek/megamek#3378]
### Environment
MegaMek map editor and standalone

### Description
I would like to be able to change the font color of terrain height, as I use exported MegaMek map images for battlemaps in other games and the black is difficult to read against most terrain. To clarify, I would like the ability to change the color of the text shown in this screenshot.
![image](https://user-images.githubusercontent.com/2125926/149795070-a9522a70-90f8-48f9-916b-cd2822c99c82.png)

If this feature already exists, you can close this issue, though I would appreciate someone pointing out how to accomplish this in that case, since I have looked through the options many times and not found it. You can already change the font color for the building height indicator, so it seems odd that I can't change this font color.

### Files
N/A

**Comment by Inglonias** [comment]
Adding the line below to my clientsettings.xml file accomplishes what I wanted:
`<preference name="MapTextColor" value="255 255 255"/>`

Leaving the issue open as I had to edit that file directly to pull this off.
